test/TEST-24-CRYPTSETUP/test.sh

```diff
@@ -39,6 +39,117 @@
     return $ret
 }
 
+can_test_pkcs11() {
+    if ! command -v "softhsm2-util" >/dev/null; then
+        ddebug "softhsm2-util not available, skipping the PKCS#11 test"
+        return 1
+    fi
+    if ! command -v "pkcs11-tool" >/dev/null; then
+        ddebug "pkcs11-tool not available, skipping the PKCS#11 test"
+        return 1
+    fi
+    if ! command -v "certtool" >/dev/null; then
+        ddebug "certtool not available, skipping the PKCS#11 test"
+        return 1
+    fi
+    if ! get_bool $(grep -F ' HAVE_P11KIT ' "${BUILD_DIR:?}/config.h"| awk '{ print $3 }'); then
+        ddebug "Support for p11-kit is disabled, skipping the PKCS#11 test"
+        return 1
+    fi
+    if ! get_bool $(grep -F ' HAVE_LIBCRYPTSETUP ' "${BUILD_DIR:?}/config.h"| awk '{ print $3 }'); then
+        ddebug "Support for libcryptsetup is disabled, skipping the PKCS#11 test"
+        return 1
+    fi
+    if ! get_bool $(grep -F ' HAVE_LIBCRYPTSETUP_PLUGINS ' "${BUILD_DIR:?}/config.h"| awk '{ print $3 }'); then
```

Comment:
I think it would make sense exposing `+LIBCRYPTSETUP_PLUGINS` in the `--version` output (see `systemd_features` in `src/basic/build.c`) and then using that for this check, similarly to the two previous cases.